src/main/java/frc/robot/commandGroups/EjectCoralThenZero.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.commandGroups;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.ElevatorCommands.ZeroElevator;
+import frc.robot.subsystems.ElevatorSubsystem;
+import frc.robot.subsystems.FunnelSubsystem;
+import frc.robot.subsystems.TootsieSlideSubsystem;
+
+public class EjectCoralThenZero extends SequentialCommandGroup {
+  public EjectCoralThenZero(ElevatorSubsystem elevatorSubsystem, TootsieSlideSubsystem tootsieSlideSubsystem, FunnelSubsystem funnelSubsystem) {
+    Command a = null;
```

Comment:
This is a bit weird, we should just declare this inline where it's used.

Reply:
Also this will add a null command to the group, probably crashes the robot code with NPE